MetadataExtractor/Formats/Apple/BplistReader.cs

```diff
@@ -43,7 +43,7 @@ public static PropertyListResults Parse(byte[] bplist)
 
         Trailer trailer = ReadTrailer();
 
-        SequentialByteArrayReader reader = new(bplist, baseIndex: checked((int)(trailer.OffsetTableOffset + trailer.TopObject)));
+        var reader = new BufferReader(bplist.AsSpan(checked((int)(trailer.OffsetTableOffset + trailer.TopObject))), isBigEndian: true);
```

Comment:
This is a nice refactoring. I'd love to see the arithmetic pulled out (this opportunity existed in the old code too):

```suggestion
        int offset = checked((int)(trailer.OffsetTableOffset + trailer.TopObject);
        var reader = new BufferReader(bplist.AsSpan(offset)), isBigEndian: true);
```